environment/directories.rst

```diff
@@ -50,6 +47,39 @@ You can use :ref:`filesystem-project` for larger datasets.
    If you also have an account on `lsst-dev <https://developer.lsst.io/services/lsst-dev.html>`_,
    you can access your Notebook Aspect home directory via :file:`$HOME/jhome/` from lsst-dev.
 
+Preinstalled subdirectories
+---------------------------
+
+Within the :file:`$HOME` directory a number of directories are currently pre-created for new users.
+
+:file:`~/notebooks`
+
+The :file:`~/notebooks` directory is intended as the top of a subtree within which you can organize your personal notebooks.
```

Comment:
Is there a way to make this sentence easier to read?

Reply:
Maybe something like:
> The `~/notebooks` directory is a place to organize notebooks.